api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java

```diff
@@ -152,10 +152,7 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
             userInfo.getEmail(), null);
     } else {
       if (user.getDisabled()) {
-        response.addHeader("disabled", "true");
-        response.setStatus(403);
-        response.sendError(HttpServletResponse.SC_FORBIDDEN, "User Disabled");
-        return false;
+        throw new ForbiddenException("This user account has been disabled.");
```

Comment:
Good to know this works. Does response.setStatus work too?

Reply:
I don't know if it does, I can check. I don't see why not though.

Reply:
It appeared to work.